### ISSUE TITLE
Use another lib and Support 1.19, 1.19.X

### ISSUE THREAD

**Issue by SoulGateKey** [Fallen-Breath/PCRC#29]
support 1.19 and 1.19.1

**Comment by Fallen-Breath** [comment]
We need to wait until the dependency [pycraft](https://github.com/ammaraskar/pyCraft) itself adds the support for 1.19+

**Comment by Fallen-Breath** [comment]
Can also check https://github.com/ammaraskar/pyCraft/issues/251

**Comment by SoulGateKey** [comment]
If it's possible to use another protocol lib https://github.com/barneygale/quarry 
~~And change it to another name QRC~~

**Comment by haorwen** [comment]
https://github.com/PrismarineJS/mineflayer This library is a protocol library that is still actively maintained and may serve as an alternative. Although this is a Node.js project, there is still a way to call it through Python: https://github.com/PrismarineJS/mineflayer/tree/master/examples/python (although this almost means completely rewriting the entire plugin).

**Comment by Fallen-Breath** [comment]
Yeah, using a lib from another programming language == totally rewrite, so not for this project
